rac_schemas/schemas/rights_statute.json

```diff
@@ -14,9 +14,9 @@
 			"title": "Statute Jurisdiction",
 			"minLength": 1,
 			"examples": [
-				"EN"
+				"en"
```

Comment:
This is a language code, not a country code 😎

Reply:
Lol, so I want to be clear with @bonniegee : we're using two-letter country codes from ISO 3166, yes? So this would be `us` instead.